engine/src/main/resources/assets/ui/menu/selectModsScreen.ui

```diff
@@ -123,7 +123,7 @@
                                     "type" : "ColumnLayout",
                                     "id" : "simpleItems",
                                     "columns" : 2,
-                                    "columnWidths" : [0.3, 0.7],
+                                    "column-widths" : [0.3, 0.7],
```

Comment:
The previous one was wrong. `ColumnLayout` defines a custom serialized name for it.